exporters/elasticsearch-exporter/src/test/java/io/zeebe/exporter/util/ElasticsearchForkedJvm.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.exporter.util;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.apache.http.HttpHost;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.RestClient;
+import org.elasticsearch.client.RestClientBuilder;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.elasticsearch.client.security.PutRoleRequest;
+import org.elasticsearch.client.security.PutUserRequest;
+import org.elasticsearch.client.security.RefreshPolicy;
+import org.elasticsearch.client.security.user.User;
+import org.elasticsearch.client.security.user.privileges.Role;
+import pl.allegro.tech.embeddedelasticsearch.EmbeddedElastic;
+import pl.allegro.tech.embeddedelasticsearch.PopularProperties;
+
+public class ElasticsearchForkedJvm implements ElasticsearchNode<ElasticsearchForkedJvm> {
+
+  private static final int DEFAULT_HTTP_PORT = 9200;
+  private static final int DEFAULT_TCP_PORT = 9300;
+  private static final String DEFAULT_VERSION = "6.6.0";
```

Comment:
This will most likely become out of sync once dependabot bumps the next elastic version. I see two options at the moment and favor the second one:

1. set a system property from maven to access the `${version.elasticsearch}` maven property https://github.com/zeebe-io/zeebe/blob/9ca84c9fa756c67333aa1598ae1a73356b196f03/parent/pom.xml#L33
1. fetch the version from the java metadata, i.e.
    ```
    ElasticsearchClient.class.getPackage().getImplementationVersion() 
    ```